jekyll/_cci2/contexts.md

```diff
@@ -117,6 +117,17 @@ jobs:
 
 If you move your repository to a new organization, you must also have the context with that unique name set in the new organization.
 
+### Contexts & Environment variables validation
+
+When creating contexts/environment variables, please note the following:
+
+- The context name must be 200 or fewer characters, must contain at least one non-whitespace character, and must not contain leading, trailing or vertical whitespaces.
+- An organization may contain no more than 500 contexts.
+- The environment variable name must be 300 or fewer characters, must have alpha or `_` as the first character and can *only* contain  alphanumeric or `_` as the remaining characters.
+- An environment variable value must be 32000 or fewer characters and must NOT contain `null`. 
+**Note**: An empty environment variable is considered valid.
+- A context may contain no more than 100 environment variables.
```

Comment:
~~ Same here, just an alternate wording that I thought might work - totally happy either way.
```suggestion
- Each context is limited to 100 environment variables.
```